README.md

```diff
@@ -11,40 +11,30 @@ The Getting Started instructions can be found [here](GettingStarted.md).
 - [Preliminary Discussion](<#preliminary discussion>)
   - [Current DEX Deficiencies](<#preliminary discussion/##Current DEX Deficiencies>)
   - [Programmable Swaps](<#preliminary discussion/##Programmable Swaps>)
-  - [cardano-swaps Protocol](<#preliminary discussion/#The cardano-swaps Protocol>)
+  - [Beacon Tokens](<#preliminary discussion/##beacon tokens>)
+  - [The Cardano-Swaps Protocol](<#preliminary discussion/#The cardano-swaps Protocol>)
 - [Specification](#Specification)
   - [Personal Contracts](<#personal contracts>)
-  - [Beacon Tokens](<#specification/##beacon tokens>)
   - [The DEX's Inline Datum](#the-dexs-inline-datum)
     - [`swapPrice`](#swapprice)
     - [`swapBeacon`](#swapbeacon)
-- [Swap Contract Logic](#the-swap-contract-logic)
-  - [Close Redeemer](#close-redeemer)
-  - [Update Redeemer](#update-redeemer)
-  - [Swap Redeemer](#swap-redeemer)
-- [Composable Atomic Swaps](#composable-atomic-swaps)
-- [Liquidity](#liquidity)
-  - [The Contrived Example](#the-contrived-example)
-  - [The Realistic Example](#the-realistic-example)
-  - [Liquidity Naturally Flows To The Less Liquid Pairs](#liquidity-naturally-flows-to-the-less-liquid-pairs)
-  - [What If Two Arbitragers Compete For The Same Swap?](#what-if-two-arbitragers-compete-for-the-same-swap)
-- [Benchmarks and Fee Estimations](#benchmarks-and-fee-estimations-ymmv)
-- [Upgradability](#upgradability)
-- [Frontend Agnosticism](<#frontend agnosticism>)
-- [Conclusion](#conclusion)
+  - [Swap Contract Logic](#the-swap-contract-logic)
+    - [Close Redeemer](#close-redeemer)
+    - [Update Redeemer](#update-redeemer)
+    - [Swap Redeemer](#swap-redeemer)
+- [Features Discussion](<#Features Discussion>)
+- [Benchmarks and Fee Estimations](<#benchmarks and fee estimations (ymmv)>)
 - [FAQ](#faq)
-  - [Why Are Reference Scripts Required?](#if-all-users-have-direct-access-to-the-spending-script-for-that-trading-pair-why-are-reference-scripts-required)
+- [Conclusion](#conclusion)
 
 ---
 ## Abstract
-`cardano-swaps` is a proof-of-concept implementation of massively scalable and concurrent DEX protocol for the Cardano Settlement Layer (CSL). It solves many of the pitfalls of current DEX implementations by empowering users to deploy their own (and interact with each others') script addresses. By doing so, users maintain fully custody of their assets at all times, and can elect if/when to upgrade their addresses to new contract standards.
+`cardano-swaps` is a proof-of-concept implementation of an *organically* scalable DEX protocol for the Cardano Settlement Layer (CSL). It solves many of the pitfalls of current DEX implementations by empowering users to deploy their own (and interact with each others') script addresses. By doing so, users always maintain spending *and* delegation custody of their assets, and can elect if/when to upgrade their addresses to new contract standards.
```

Comment:
"spending *and* delegation custody" sounds awkward here. People don't usually say "spending custody" or "delegation custody". I think "custody" should be replaced with "control".